app/controllers/user_controller.rb

```diff
@@ -12,6 +12,10 @@ def post
 
     requester_email = session.fetch('email')
     email_list = @form.email_list
+    if email_list.any? { |email| email.length > 64 } then
+      @form.errors.add 'email_list', 'contains email address over 64 characters in length - see if you can get the user an alias'
+      return render :user
+    end
```

Comment:
I think `@form.invalid?` will be true if errors are added, so you could maybe move line 11 down, and then remove line 17. That might give the user multiple errors rather than just one error at a time.